### PR TITLE
Replace color issue fix for #240

### PR DESCRIPTION
**Pull request by christopherbauer**
I think this code may have been copy pasted in from elsewhere which might explain why instead of performing a replacement it appears to be trying to determine the difference between the origin pixels.

### CODE REVIEW

File: src/ImageProcessor.Playground/Program.cs [hunk 1]
```diff
@@ -35,48 +33,12 @@ public class Program
...
-
-                // Report back.
-                long peakWorkingSet64 = Process.GetCurrentProcess().PeakWorkingSet64;
-                float mB = peakWorkingSet64 / (float)1024 / 1024;
-
-                Console.WriteLine(@"Completed {0} in {1:s\.fff} secs {2}Peak memory usage was {3} bytes or {4} Mb.", fileInfo.Name, stopwatch.Elapsed, Environment.NewLine, peakWorkingSet64.ToString("#,#"), mB);
-            }
-
-            Console.ReadLine();
+            var files = GetFilesByExtensions(new DirectoryInfo(Environment.CurrentDirectory).GetDirectories("Images").Single(), ".png");
```
Comment (JimBobSquarePants) [inline src/ImageProcessor.Playground/Program.cs]: Can we make sure the playground code is not altered on merge? The timing and memory diagnostics code is very useful here.

**Comment by JimBobSquarePants** [comment]
Can you post the transformed test image running this code?

**Comment by christopherbauer** [comment]
test

File: src/ImageProcessor.Playground/Program.cs (hunk 1 above)
Comment (christopherbauer) [inline src/ImageProcessor.Playground/Program.cs]: Yeah I'll roll that back in the branch when the fix is considered complete.

**Comment by christopherbauer** [comment]
Hmm looking at the two images, that snippet I killed was doing more than I thought. I'll have to take a second look.

**Pull request closed without merging**